src/brands/kate-spade/default/font/face.json

```
@@ -4,7 +4,7 @@
       "1": {
         "normal": {
           "family": {
-            "value": "FuturaPT-Book"
+            "value": "Futura Now Var Regular"
```

To match the convention already in place, can we make this value match to the woff2 file name?

```suggestion
            "value": "FuturaNowTextRegular"
```